package-lock.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
As indicated in the single commit, this is dead code and should be removed. It caused me confusion when initially setting up the project.